code/firmware/rosco_m68k_firmware/main1.c

```diff
@@ -230,10 +230,16 @@ if (!have_video) {
     print_cpu_mem_info();
 
 #ifdef BLOCKDEV_SUPPORT
+#ifdef ROSCO_M68K_ATA
     ata_init();
+#endif
     INSTALL_BLOCKDEV_HANDLERS();
 #endif
 
+    // Reset the SDB_STATUS, since we (ab)use it for various things during hardware init...
+    uint32_t *sdb_status = (uint32_t*)0x404;
+    *sdb_status = 0xC001C001;
+
```

Comment:
I could see re-using SDB_STATUS being confusing if the probing ever gets used in the future after this point. Your comment at the declaration of `INSTALL_TEMP_BERR_HANDLER_C` does address not using it beyond early init, but it would be nice for the firmware to be still able to probe later if the need arose.

Reply:
This point is more minor, I don't care as much about it as the `INSTALL_TEMP_BERR_HANDLER_C`, it's just a mild preference/opinion.

Reply:
I think the rules about not probing later are probably enough, and if the need arose to probe later we should fix it at that point, but agree using `SDB_STATUS` for this was a bit of a hack, cleaned it up by giving it a dword in `.early_data` instead.